elements/lisk-p2p/src/peer_directory/new_peers.ts

```diff
@@ -12,193 +12,48 @@
  * Removal or modification of this copyright notice is prohibited.
```

Comment:
I propose renaming this and `tried_peers` to `new_list` and `tried_list` to stay in conjunction with `base_list`

Reply:
I like this proposal. I'd go even further because `new` is not exactly what we have since we move back `tried` peers to `new`. Could we find a better adjective for `new` peers?